src/parse.rs

```diff
@@ -307,19 +307,23 @@ pub(crate) async fn handle_unilateral(
                         })
                         .collect(),
                 })
-                .await;
+                .await; //TODO: decide what to do with result
```

Comment:
This can only error if the receiver is dropped, and the receiver is stored in the same Session struct as the sender, so you can no drop one without dropping the other.  So I think it's fine to ignore the result and call `.ok()` on it so something like that.

Reply:
It should at least print a warning, or `expect`, otherwise it is easy to hide a subtle error if sth is not correct in the setup